pkg/virt-handler/node-labeller/cpu_plugin.go

```diff
@@ -150,8 +151,12 @@ func (n *NodeLabeller) loadHostSupportedFeatures() error {
 
 func (n *NodeLabeller) loadHostCapabilities() error {
 	capsFile := filepath.Join(n.volumePath, "capabilities.xml")
-	n.capabilities = &api.Capabilities{}
-	err := n.getStructureFromXMLFile(capsFile, n.capabilities)
+	n.capabilities = &libvirtxml.Caps{}
+	rawFile, err := os.ReadFile(capsFile)
+	if err != nil {
+		return err
+	}
+	err = n.capabilities.Unmarshal(string(rawFile))
```

Comment:
To use the `Unmarshal` method of libvirtxml `Caps` would require me to alter `getStructureFromXMLFile` in a way that would obstruct its other uses (e.g. `getDomCapabilities`).

I think the general direction for `cpu_plugin.go` as a whole is towards complete deprecation in favor of construction using basic types (the necessary ones) and inference at the virt-handler level. Should it prove its use, it may be worth it to have `cpu_plugin.go` as its own separate package with the responsibility of obtaining said information.